mlir/lib/Mitigation/Transforms/MitigationMethods/Zne.cpp

```diff
@@ -45,9 +45,12 @@ void ZneLowering::rewrite(mitigation::ZneOp op, PatternRewriter &rewriter) const
     RankedTensorType scaleFactorType = cast<RankedTensorType>(scaleFactors.getType());
     const auto sizeInt = scaleFactorType.getDimSize(0);
 
+    // Folding type
+    auto foldingAlgorithm = op.getFolding();
+
     // Create the folded circuit function
     FlatSymbolRefAttr foldedCircuitRefAttr =
-        getOrInsertFoldedCircuit(loc, rewriter, op, scaleFactorType.getElementType());
```

Comment:
how come `scaleFactorType.getElementType()` is no longer needed?

Reply:
Check the method signature. It does match, and global folding continues to work, but I'm not sure exactly where or why this happened.

Reply:
🤷 maybe it was some leftover from old code

Reply:
I probably forgot to remove it and it is unused

Reply:
That seems likely. I think we can consider this "resolved," then.